content/source/docs/extend/testing/acceptance-tests/teststep.html.md

```diff
@@ -287,5 +287,6 @@ func testAccCheckExampleWidgetExists(resourceName string, widget *example.Widget
 }
 ```
 
+## Next Steps
 
-[1]: /docs/extend/testing/acceptance-testing/bestpractices.html
+Acceptance Testing is an essential approach to validating the implementation of a Terraform Provider. Unfortunately using actual APIs to provision resources for testing can leave behind real infrastructure that costs money between tests. The reasons for these leaks can vary, regardless Terraform provides a mechanism know as [Sweepers](/docs/extend/testing/acceptance-tests/sweepers.html) to help keep the vendor testing account clean.
```

Comment:
Nitpick: I'm not sure if _Unfortunately_ is really necessary here, as it's just summary 🤔

_to help keep the vendor testing account clean_ -> _to help keep the testing account clean_ - Really nitpicky, but "vendor" can be a bit confusing, depending on the context (owner, provider, payer of the account). I'd just leave it out.